### ISSUE TITLE
RuntimeError: maximum recursion depth exceeded while calling a Python object

### ISSUE THREAD

**Issue by odex** [nilp0inter/binlog#9]
After the updating of binlog v5.0.0, I have migrated my large db and when I try to use the "purge" method this causes an exception:

```
  ...
  File ".../site-packages/binlog/operations.py", line 24, in __next__
    a_val = next(self.a)
  File "/.../site-packages/binlog/operations.py", line 24, in __next__
    a_val = next(self.a)
RuntimeError: maximum recursion depth exceeded while calling a Python object
```

Python version: 3.4

**Comment by nilp0inter** [comment]
Please attach the beginning of the stack trace.

**Comment by odex** [comment]
~2500

**Comment by nilp0inter** [comment]
It should be fixed in v5.1.0. Incrementing the recursion limit is now not necessary.
